pkg/propertyprovider/azure/cloudconfig/config.go

```diff
@@ -0,0 +1,150 @@
+package cloudconfig
```

Comment:
Split this cloud config change in a separate PR. Add a comment before this package. Move this package to /pkg/utils/cloudconfig/azure.

Reply:
Separate PR: https://github.com/Azure/fleet/pull/943